### PR TITLE
Update brew-php-switcher to version 1.4

### PR DESCRIPTION
**Pull request by philcook**

### CODE REVIEW

**Comment by mirfilip** [comment]
@philcook skip option for Apache is a very useful addition, could you please help this pull request to go through ?

Moreover, are you sure `SHA1` hash is correct ? When you try to install **brew-php-switcher** from this pull request, you get:
```
> brew install https://raw.githubusercontent.com/philcook/homebrew-php/brew-php-switcher-1.4/Formula/brew-php-switcher.rb
(...)
Error: SHA1 mismatch
Expected: 48BBF98F2D55D0B92D7FBA28D43C775876432293
Actual: 48bbf98f2d55d0b92d7fba28d43c775876432293
```

Hash seems the same, just chars case differs - looks suspicious.

**Comment by danielcosta** [comment]
Unfortunatelly the build log is not present anymore on Jenkins, but really this PR is very useful

**Comment by philcook** [comment]
Hey guys,

Updated from sha1 to sha256 as advised here
https://github.com/Homebrew/homebrew/blob/master/Library/Contributions/example-formula.rb

Let me know if you need anything else.

File: Formula/brew-php-switcher.rb [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
-  head 'https://github.com/philcook/brew-php-switcher.git'
+  url "https://github.com/philcook/brew-php-switcher/archive/v1.4.tar.gz"
+  homepage "https://github.com/philcook/php-switcher"
+  sha256 "299950a2d3cfa70468aa9569f10c8d50eff68ec069c7a884898677e6799932bb"
+  version "1.4"
+
+  stable do
+    url "https://example.com/foo-1.0.tar.gz"
+    sha256 "2a2ba417eebaadcb4418ee7b12fe2998f26d6e6f7fda7983412ff66a741ab6f7"
+  end
```
Comment (alanthing) [inline Formula/brew-php-switcher.rb]: Please remove this `stable` block with invalid URL.

File: Formula/brew-php-switcher.rb [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
+
+  head "https://github.com/philcook/brew-php-switcher.git"
+  head do
+    url "https://github.com/philcook/brew-php-switcher.git"
+  end
```
Comment (alanthing) [inline Formula/brew-php-switcher.rb]: Please remove one of these duplicate `head` blocks.

**Comment by alanthing** [comment]
@philcook I left some comments for changes you should make. Then, please squash your commits down to a single commit and re-push.

**Comment by philcook** [comment]
@alanthing thanks for your feedback. Done as requested regarding removing stable and reducing head and now a single commit.

**Comment by philcook** [comment]
@alanthing the build was constantly failing so I thought I was doing something wrong here. Is it just the build tool is broken?

**Comment by alanthing** [comment]
It's a little touchy. I reviewed failed build logs and will push some through if the failures are due to build bot errors. Let's see how this one turns out. Thanks for updating the PR.

**Comment by alanthing** [comment]
Looks good @philcook. Check out df1fedd as I made some changes and melded them into your commit. I added a test, I removed the version (homebrew can figure it out from the URL), and I flattened the head block. You may want to grab a copy for future changes.

**Comment by philcook** [comment]
Wonderful @alanthing thanks for your help and glad it got through.
Will take latest copy of homebrew-php so don't have this problem next time.

**Pull request closed without merging**